deploy/Chart/templates/controller/deployment.yaml

```diff
@@ -72,7 +72,23 @@ spec:
           mountPath: {{ .Values.global.rootCA.mountPath }}
           readOnly: true
         {{- end }}
+        - name: work-dir
```

Comment:
opened a PR here for the bicep container work: https://github.com/radius-project/radius/pull/8002